src/components/AddAsset/AddAssetComponent.jsx

```diff
@@ -0,0 +1,152 @@
+import PropTypes from 'prop-types';
+import * as React from 'react';
+import {
+  Button,
+  Form,
+  Input,
+  Header,
+  Grid
+} from 'semantic-ui-react';
+
+import AddAssetDropdown from './AddAssetDropdown';
+import SideMenuComponent from '../../_components/SideMenuComponent';
+import { SemanticToastContainer } from 'react-semantic-toasts';
```

Comment:
Absolute imports should come before relative imports, therefore, move this before `AddAssetDropdown` import